.github/workflows/flaky-test-debug.yml

```diff
@@ -0,0 +1,45 @@
+name: Flaky Test Debug
```

Comment:
Are we planning to keep this around on `master` for future flaky test debugging branches?

Reply:
Yes we are - that's why I left it in. Would just need to customize it to the specific flaky test but it would allow us to create many CI runs in a row for easier testing.